Driver_Mutation_Results/workflow/steps/driver_mutation_results/calculate_driver_results.cwl

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env cwl-runner
+#
+# Authors: Bruno Grande
```

Comment:
```suggestion
```

Let's get rid of these lines, since there is the metadata at the bottom.(I think the example should have this removed as well.)